cypress/integration/manualTests/basic-enduser.ts

```diff
@@ -0,0 +1,178 @@
+import { Dockstore } from '../../../src/app/shared/dockstore.model';
+import { goToTab } from '../../support/commands';
+
+function testWorkflow(url: string, version1: string, version2: string, trsUrl: string) {
+  it('get the svg icons', () => {
+    cy.visit('/workflows/' + url + ':' + version1);
+    cy.get('[data-cy=dnanexusIcon]').within(() => {
+      cy.get('svg').should('exist');
+    });
+    cy.get('[data-cy=terraIcon]').within(() => {
+      cy.get('svg').should('exist');
+    });
+    cy.get('[data-cy=anvilIcon]').within(() => {
+      cy.get('svg').should('exist');
+    });
+  });
+
+  it('info tab works', () => {
+    goToTab('Launch');
+    cy.url().should('contain', '?tab=launch');
+    cy.contains('mat-card-header', 'Workflow Information');
+    // test export to zip button?
+  });
+
+  it('versions tab works', () => {
+    goToTab('Versions');
+    cy.url().should('contain', '?tab=versions');
+
+    // check that clicking on a different version goes to that version's url
+    cy.contains('[data-cy=versionName]', version2).click();
+    cy.url().should('contain', url + ':' + version2);
+  });
+
+  it('files tab works', () => {
```

Comment:
Can you check that the two sub tabs can be clicked and if there is content within them?